config_bucket.go

```diff
@@ -0,0 +1,72 @@
+package evergreen
+
+import (
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const (
+	BucketTypeLocal = "local"
+	BucketTypeS3    = "s3"
+)
+
+// BucketConfig represents the admin config section for bucket storage.
+type BucketConfig struct {
+	LogBucket     string `bson:"log_bucket" json:"log_bucket" yaml:"log_bucket"`
```

Comment:
could it make sense to make the bucket config a struct of structs? Like
```go
type BucketConfig struct {
    LoggingBucket Bucket
}

type Bucket struct {
    Name string
    Type string
}
```
(to facilitate adding more buckets)